The job **`x86_64-gnu-tools`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/3795084457/jobs/6453833636) [(plain)](https://github.com/rust-lang/rust/commit/c2fdd396d36526a3dadef6b70308a6740d749cb1/checks/10336931383/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
  IMAGE: x86_64-gnu-tools
##[endgroup]
From https://github.com/rust-lang/rust
 * branch              master     -> FETCH_HEAD
Searching for toolstate changes between 6a20f7df5755d8c6b68110d2d0391a7b03268e77 and 07d83640dbaa1fe180806b008d1ba26d7f429f68
Rustdoc was updated
##[group]Run src/ci/scripts/verify-channel.sh
src/ci/scripts/verify-channel.sh
shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
env:
---
.......... (90/96)
......     (96/96)


/checkout/src/test/rustdoc-gui/codeblock-tooltip.goml codeblock-tooltip... FAILED
[ERROR] (line 25 from 135) Error: Evaluation failed: The following errors happened (for selector `.docblock .example-wrap.compile_fail .tooltip`): [expected `center` for key `text-align`, found `start`]: for command `assert-css: (
            ".docblock .example-wrap.compile_fail .tooltip::after",
            {
                "content": '"This example deliberately fails to compile"',
                "text-align": "center",
                "padding": "5px 3px 3px",
                "background-color": |background|,
                "color": |color|,
                "border": "1px solid " + |border|,
        )`
        )`
[ERROR] (line 65 from 135) Error: Evaluation failed: The following errors happened (for selector `.docblock .example-wrap.should_panic .tooltip`): [expected `center` for key `text-align`, found `start`]: for command `assert-css: (
            ".docblock .example-wrap.should_panic .tooltip::after",
            {
                "content": '"This example panics"',
                "text-align": "center",
                "padding": "5px 3px 3px",
                "background-color": |background|,
                "color": |color|,
                "border": "1px solid " + |border|,
        )`
        )`
[ERROR] (line 105 from 135) Error: Evaluation failed: The following errors happened (for selector `.docblock .example-wrap.ignore .tooltip`): [expected `center` for key `text-align`, found `start`]: for command `assert-css: (
            ".docblock .example-wrap.ignore .tooltip::after",
            {
                "content": '"This example is not tested"',
                "text-align": "center",
                "padding": "5px 3px 3px",
                "background-color": |background|,
                "color": |color|,
                "border": "1px solid " + |border|,
        )`
        )`
[ERROR] (line 25 from 141) Error: Evaluation failed: The following errors happened (for selector `.docblock .example-wrap.compile_fail .tooltip`): [expected `center` for key `text-align`, found `start`]: for command `assert-css: (
            ".docblock .example-wrap.compile_fail .tooltip::after",
            {
                "content": '"This example deliberately fails to compile"',
                "text-align": "center",
                "padding": "5px 3px 3px",
                "background-color": |background|,
                "color": |color|,
                "border": "1px solid " + |border|,
        )`
        )`
[ERROR] (line 65 from 141) Error: Evaluation failed: The following errors happened (for selector `.docblock .example-wrap.should_panic .tooltip`): [expected `center` for key `text-align`, found `start`]: for command `assert-css: (
            ".docblock .example-wrap.should_panic .tooltip::after",
            {
                "content": '"This example panics"',
                "text-align": "center",
                "padding": "5px 3px 3px",
                "background-color": |background|,
                "color": |color|,
                "border": "1px solid " + |border|,
        )`
        )`
[ERROR] (line 105 from 141) Error: Evaluation failed: The following errors happened (for selector `.docblock .example-wrap.ignore .tooltip`): [expected `center` for key `text-align`, found `start`]: for command `assert-css: (
            ".docblock .example-wrap.ignore .tooltip::after",
            {
                "content": '"This example is not tested"',
                "text-align": "center",
                "padding": "5px 3px 3px",
                "background-color": |background|,
                "color": |color|,
                "border": "1px solid " + |border|,
        )`
        )`
[ERROR] (line 25 from 147) Error: Evaluation failed: The following errors happened (for selector `.docblock .example-wrap.compile_fail .tooltip`): [expected `center` for key `text-align`, found `start`]: for command `assert-css: (
            ".docblock .example-wrap.compile_fail .tooltip::after",
            {
                "content": '"This example deliberately fails to compile"',
                "text-align": "center",
                "padding": "5px 3px 3px",
                "background-color": |background|,
                "color": |color|,
                "border": "1px solid " + |border|,
        )`
        )`
[ERROR] (line 65 from 147) Error: Evaluation failed: The following errors happened (for selector `.docblock .example-wrap.should_panic .tooltip`): [expected `center` for key `text-align`, found `start`]: for command `assert-css: (
            ".docblock .example-wrap.should_panic .tooltip::after",
            {
                "content": '"This example panics"',
                "text-align": "center",
                "padding": "5px 3px 3px",
                "background-color": |background|,
                "color": |color|,
                "border": "1px solid " + |border|,
        )`
        )`
[ERROR] (line 105 from 147) Error: Evaluation failed: The following errors happened (for selector `.docblock .example-wrap.ignore .tooltip`): [expected `center` for key `text-align`, found `start`]: for command `assert-css: (
            ".docblock .example-wrap.ignore .tooltip::after",
            {
                "content": '"This example is not tested"',
                "text-align": "center",
                "padding": "5px 3px 3px",
                "background-color": |background|,
                "color": |color|,
                "border": "1px solid " + |border|,
        )`

Build completed unsuccessfully in 0:02:34
```

</details>

--- a/src/librustdoc/html/static/css/rustdoc.css
+++ b/src/librustdoc/html/static/css/rustdoc.css
@@ -1109,7 +1109,6 @@ pre.rust .doccomment {
 }
 
 .example-wrap .tooltip:hover::after {
-	text-align: center;
 	padding: 5px 3px 3px 3px;
 	border-radius: 6px;
 	margin-left: 5px;

--- a/src/test/rustdoc-gui/codeblock-tooltip.goml
+++ b/src/test/rustdoc-gui/codeblock-tooltip.goml
@@ -34,7 +34,6 @@ define-function: (
             ".docblock .example-wrap.compile_fail .tooltip::after",
             {
                 "content": '"This example deliberately fails to compile"',
-                "text-align": "center",
                 "padding": "5px 3px 3px",
                 "background-color": |background|,
                 "color": |color|,
@@ -74,7 +73,6 @@ define-function: (
             ".docblock .example-wrap.should_panic .tooltip::after",
             {
                 "content": '"This example panics"',
-                "text-align": "center",
                 "padding": "5px 3px 3px",
                 "background-color": |background|,
                 "color": |color|,
@@ -114,7 +112,6 @@ define-function: (
             ".docblock .example-wrap.ignore .tooltip::after",
             {
                 "content": '"This example is not tested"',
-                "text-align": "center",
                 "padding": "5px 3px 3px",
                 "background-color": |background|,
                 "color": |color|,

